src/components/Contacts/ContactDetails/ContactDetailsTab/Mailing/ContactDetailsTabMailing.stories.tsx

```diff
@@ -0,0 +1,29 @@
+import { Box, CssBaseline, MuiThemeProvider } from '@material-ui/core';
+import React, { ReactElement } from 'react';
+import { gqlMock } from '../../../../../../__tests__/util/graphqlMocking';
+import theme from '../../../../../theme';
+import {
+  ContactDetailsTabDocument,
+  ContactDetailsTabQuery,
+  ContactDetailsTabQueryVariables,
+} from '../ContactDetailsTab.generated';
+import { ContactDetailsTabMailing } from './ContactDetailsTabMailing';
+
+export default {
+  title: 'Contacts/Tab/ContactDetailsTab/Mailing',
+};
+
+export const Default = (): ReactElement => {
+  const mock = gqlMock<ContactDetailsTabQuery, ContactDetailsTabQueryVariables>(
+    ContactDetailsTabDocument,
+    { variables: { accountListId: '111', contactId: '222' } },
+  );
+  return (
+    <Box m={2}>
+      <MuiThemeProvider theme={theme}>
+        <CssBaseline />
```

Comment:
this should already be applied to all stories via: https://github.com/CruGlobal/mpdx-react/blob/main/.storybook/preview.js#L31-L42

Reply:
@frett I don't think it is working because in the code below if I remove the ThemeProvider I don't get the correct style.

https://github.com/CruGlobal/mpdx-react/blob/27b7990c748ebb04e3258a35fb00cdad203944d8/src/components/Contacts/ContactDetails/ContactDetailsTab/Tags/ContactTags.stories.tsx#L29

Reply:
I added a box with a background color of red to `preview.js` and it changed the background color of the preview area to red. so it is definitely processing it, not sure if something else is messing with it

Reply:
I bet it needs to be a `MuiThemeProvider` and not just a `ThemeProvider`

Reply:
OK, I will remove the theme not sure why my test environment was not showing the theme at first but it currently is working.